src/apps/loyalty/server/index.tsx

```diff
@@ -69,6 +70,18 @@ app.get("/inquiries", (req, res) => {
     return res.redirect(req.baseUrl + "/login")
   }
 
+  const headers = { "X-Access-Token": req.user.get("accessToken") }
```

Comment:
Ah ok, that makes sense. we can still move that gravity call to a middleware though.
that way you can just check `res.locals.user.confirmed_buyer_at` if the user is logged in, in any route.